core/app/views/system_notifications/organization-invite-existing-user.text.erb

```diff
@@ -1,13 +0,0 @@
-Hi there
-=================================================================
-
-You have been invited by <%= @info[:ref_full_name] %> to join <%= @info[:organization] %> on our platform!. You can automatically create your account on our platform and join <%= @info[:organization] %> by clicking on the link below:
-<%= @info[:confirmation_link] %>
-
-<% if @info[:team] %>
-By accepting this invite you will automatically join the <%= @info[:team] %> team.
-<% end %>
-
-
-Regards,
-The Marketplace team
```

Comment:
Did you delete this file? I think it should still be there?

Reply:
The file got re-added as a new file. https://github.com/maestrano/mno-enterprise/pull/631/files#diff-d649e7a287c0118e8319da0b025b4790R1